opflexagent/gbp_ovs_agent.py

```diff
@@ -112,6 +123,20 @@ def __init__(self, root_helper=None, **kwargs):
         # TODO(ivar): make this component pluggable.
         self.ep_manager = ep_manager.EndpointFileManager().initialize(
             self.host, self.bridge_manager, kwargs)
+        self.init_firewall()
+
+    def _invoke_ep_mgr(self, port):
```

Comment:
hypervisor_type is already a user configurable parameter, we could use it to chose which manager to use.